brewtils/plugin.py

```diff
@@ -214,6 +212,10 @@ def __init__(self, client=None, system=None, logger=None, **kwargs):
         self._system = self._setup_system(system, kwargs)
         self._ez_client = EasyClient(logger=self._logger, **self._config)
 
+        # Make sure this is set after _system
+        if client:
+            self.client = client
+
```

Comment:
Should this be `self._client = client` ?

Reply:
I need the code that's defined in the client setter function to run:

```python
@client.setter
def client(self, new_client):
    ...
```

Setting `client` instead of `_client` is the magic that makes that happen.